alf/algorithms/mcts_algorithm.py

```diff
@@ -232,6 +243,17 @@ def __init__(
                 shape as ``steps``.
             reward_spec (TensorSpec): a rank-1 or rank-0 tensor spec representing
                 the reward(s).
+            expand_all_children (bool): If True, when a new leaf is selected, immediately
+                expand all its children.
+            expand_all_root_children (bool): whether to expand all root children
+                before search. This is described in Appendix A of "Learning and
+                Planning in Complex Action Spaces". However, our implementation is
+                different from the paper's. The paper initializes Q(s, a) for
+                root s for all the action being sampled. We expand all sampled
+                action for root s. Our implementation depends on setting
+                (act/learn)_with_exploration_policy=True as the visit counts
```

Comment:
Does this mean we need to set `(act/learn)_with_exploration_policy=True` when `expand_all_root_children` is set as `True`?  If this is the case, it might be useful to added a check on this condition to avoid mis-use.